setup.py

```diff
@@ -18,7 +18,7 @@
 
 setup(
     name="confidant-client",
-    version="2.7.1",
+    version="2.8.0",
```

Comment:
I also wouldn't bump the version here. We're no longer going to upload to PyPI so this `2.8.0` version won't be published.